src/overlays/actors/ovl_En_Sth2/z_en_sth2.h

```diff
@@ -5,9 +5,18 @@
 
 struct EnSth2;
 
+typedef void (*EnSth2ActionFunc)(struct EnSth2*, PlayState*);
+
 typedef struct EnSth2 {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x10C];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable;
+    /* 0x18E */ UNK_TYPE1 pad18E[0x5A];
+    /* 0x1E8 */ Vec3s morphTable;
+    /* 0x1EE */ UNK_TYPE1 pad1EE[0x5A];
```

Comment:
These joint/morph tables are arrays of length equal to your skeleton limb, which is where the padding comes in.
you should be able to replace them with something like 
```suggestion
    /* 0x188 */ Vec3s jointTable[OBJECT_STH_LIMB_MAX];
    /* 0x1E8 */ Vec3s morphTable[OBJECT_STH_LIMB_MAX];
```
well.. except those are really long tables, if the skeleton doesnt have that much you will still some _some_ padding I guess.